app/assets/stylesheets/moderators.scss

```diff
@@ -155,8 +159,446 @@
   padding: 30px 10px;
 }
 
+// these styles are to correct height and margin issues in the iframe
+.moderations-actions_panel {
+  min-height: initial;
+  height: 100%;
+  #page-content-inner {
+    margin-bottom: 0;
+    height: 100%;
+  }
+}
+
 .mod-container {
-  padding: 30px 0px;
+  display: flex;
+  flex-direction: column;
+  background-color: var(--base-0);
+  border: none;
+  box-shadow: none;
```

Comment:
Is there another CSS class in a parent element that has `border` and `box-shadow` set which is why you need to set them to `none`?

Reply:
Hmm I needed this in the beginning but after testing with the latest changes maybe I don't... Will revisit.